app/src/main/java/com/example/footprnt/Profile/EditPost.java

```diff
@@ -120,6 +100,50 @@ public void done(ParseException e) {
             }
         });
 
+
+        mBtnSave.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                post.setTitle(mEtTitle.getText().toString());
+                post.setDescription(mEtDescription.getText().toString());
+                post.getUser().saveInBackground(new SaveCallback() {
+                    @Override
+                    public void done(ParseException e) {
+                        Intent it = new Intent();
+                        it.putExtra("position", position);
+                        setResult(301, it);
+                        finish();
+                    }
+                });
+
+            }
+        });
+
     }
 
+
+    private void setViews(){
+        String date = util.getRelativeTimeAgo(post.getCreatedAt().toString());
+        mTvDate.setText(date);
+        mEtTitle.setText(post.getTitle());
+        mEtDescription.setText(post.getDescription());
+
+        StringBuilder sb = new StringBuilder();
+        if(post.getCity()!=null){
+            sb.append(post.getCity()+",");
```

Comment:
I'd prefer to see
  sb.append(post.getCity() + ",");
but I'd love to see
  sb.append(post.getCity().append(",");